internal/pkg/template/templates/cicd/pipeline_cfn.yml

```diff
@@ -236,6 +238,21 @@ Resources:
                 Provider: Manual
               RunOrder: {{$stage.Approval.RunOrder}}
             {{- end}}
+            {{- range $action := $stage.PreDeployments }}
```

Comment:
Why do we only add preDeployments here? What about postDeployments?

Reply:
In line 257 👍🏼 .